src/main/kotlin/step2/util/OperationUtil.kt

```diff
@@ -0,0 +1,17 @@
+package step2.util
+
+class OperationUtil {
+    companion object {
+        private val operators = setOf("+", "-", "*", "/")
```

Comment:
이부분, 뭔가 enum으로 바꿔쓰고싶었는데 
이 util 자체가 스트링에서 연산자, 숫자를 뽑을 때 사용하는 거라 enum으로 변경하기 애매해서 놔뒀습니다.

추가 연산자가 생긴다면 OperatorType enum 뿐 아니라 여기서도 수정해야하는 불상사가..

Reply:
```suggestion
        private val operators = OperatorType.values().map(OperatorType::name)
```
요렇게 사용하려다가 놔두신건가요?? ㅎㅎ

Reply:
오 경현님 최고의 코멘트..!

Reply:
저걸 왜 생각 못했지 바로 반영했습니다 ㅋㅋㅋ